game/scripts/npc/abilities/crystal_maiden_frostbite.txt

```diff
@@ -44,12 +44,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "total_damage"                                    "150 200 250 300 350 450"
+        "total_damage"                                    "150 200 250 300 350 675"
       }
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "creep_total_damage"                              "1000"
+        "creep_total_damage"                              "1000 2000 3000 4000 5000 6000" //OAA
```

Comment:
Lmao, wtf is this? Farming CM? xD Wiki says this: Illusions, creep-heroes and Roshan are treated as heroes. So if you wanted to make Frostbite viable against bosses, this will not work.

Reply:
ye i realize that, it also doesn't work vs black dragons either but hey it might be helpful 🤷‍♂️

Reply:
Change creep duration too then.